src/app/app.element.ts

```diff
@@ -31,7 +29,7 @@ export class UmbAppElement extends UmbLitElement {
 	 * @remarks This is the base URL of the Umbraco server, not the base URL of the backoffice.
 	 */
 	@property({ type: String })
-	private serverUrl = import.meta.env.VITE_UMBRACO_API_URL ?? '';
+	serverUrl? = window.location.origin;
```

Comment:
I would say the default should just keep being `''` since it is being prepended to API calls and we probably just want to keep them relative in CMS land.

```suggestion
	serverUrl = '';
```